meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -488,7 +488,9 @@ board_config() {
       BIOS_LINK_COMM="$FW_STORE_URL/$DASHARO_REL_NAME/v$DASHARO_REL_VER.rom"
       HAVE_EC="false"
       NEED_EC_RESET="false"
-      BIOS_HASH_LINK_COMM="a880504dfb0497f31e898f62ae5c9f3b145bca0b4fa601e41cddc54bea22ee36  $BIOS_UPDATE_FILE"
+      #BIOS_HASH_LINK_COMM="a880504dfb0497f31e898f62ae5c9f3b145bca0b4fa601e41cddc54bea22ee36  $BIOS_UPDATE_FILE"
```

Comment:
We can simply remove that line, the hash is validated against the signature anyways.

Reply:
```suggestion
```

Reply:
Thanks for confirming.  I just pushed a commit to remove that commented line.